src/sql/workbench/contrib/query/browser/statusBarItems.ts

```diff
@@ -289,7 +292,7 @@ export class QueryResultSelectionSummaryStatusBarContribution extends Disposable
 
 	private onCellSelectionChanged(data: string[]): void {
 		const numericValues = data?.filter(value => !Number.isNaN(Number(value))).map(value => Number(value));
-		if (numericValues?.length < 2 || !(this.editorService.activeEditor instanceof QueryEditorInput)) {
+		if (numericValues?.length < 2) {
```

Comment:
So does this already handle displaying the status bar item for Notebook editors with the input check being removed? And then you just also added the call to hide it when the query is started?

Reply:
Yep I believe the input check was added temporarily to disable the select summary status bar for notebooks. Is that correct @alanrenmsft?